presto-docs/src/main/sphinx/connector/jmx.rst

```diff
@@ -18,21 +21,35 @@ To configure the JMX connector, create a catalog properties file
 
     connector.name=jmx
 
+To enable periodical dumps, define following properties:
+
+.. code-block:: none
+
+    jmx.dump-tables=java.lang:type=Runtime,com.facebook.presto.execution.scheduler:name=NodeScheduler
+    jmx.dump-period=10s
+    jmx.max-entries=86400
+
+``dump-tables`` is a comma separated list of Managed Beans (MBean). It specifies
+which MBeans will be sampled and stored in memory every ``dump-period`` milliseconds.
```

Comment:
dump-period takes an interval with arbitrary unit, so "milliseconds" is incorrect.